scripts/evaluate.py

```diff
@@ -78,6 +86,8 @@ def parse_args():
     parser.add_argument('detection', type=str, help='Text detection model to use for analysis')
     parser.add_argument('recognition', type=str, help='Text recognition model to use for analysis')
     parser.add_argument('--iou', type=float, default=0.5, help='IoU threshold to match a pair of boxes')
+    parser.add_argument('-dataset', type=str, default='FUNSD', help='choose a dataset: FUNSD, CORD')
+    parser.add_argument('-path', type=str, default=None, help='Only for local sets, path to your OCRDataset')
```

Comment:
If those are optional args, we need two hyphens (one hyphen only if that's the shortcut arg with a single character)